server/migrations/README.md

```diff
@@ -0,0 +1,3 @@
+https://www.npmjs.com/package/mongo-migrate
```

Comment:
Should this file be here?

Reply:
you told me to put a readme in there, how do I link that to the main readme

Reply:
[Use Google](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)